.github/dependabot.yml

```diff
@@ -13,6 +13,11 @@ updates:
     directory: /
     schedule:
       interval: weekly
+    ignore:
+      # Ignore the following modules as they are updated by k8s.io/kubectl
```

Comment:
imo, this makes sense if we can prove that all the releases for these modules are ALWAYS triggered together

Reply:
we cannot, but in most cases, you don't need to update them immediately after release.